docs/conventions.md

```diff
@@ -12,15 +12,26 @@ corresponding terms in VSCode:
 | window | editor group |
 | frame  | window       |
 
-### Formatting in Major Mode
+## Common major mode key bindings
 
-The following formatting bindings should be offered in a major mode `<spc> m`:
+The following formatting bindings should be offered in all major modes, i.e. `<spc> m`:
```

Comment:
```suggestion
The following formatting bindings should be offered in all major modes that uses language server protocol.
```
Nit: Not sure if we need to mention it or not. I am thinking some major mode does not need these e.g. markdown.